dist/mentio.js

```diff
@@ -463,7 +465,8 @@ angular.module('mentio', [])
                 items: '=mentioItems',
                 triggerChar: '=mentioTriggerChar',
                 forElem: '=mentioFor',
-                parentScope: '=mentioParentScope'
+                parentScope: '=mentioParentScope',
+                zIndex: '@mentioZIndex'
```

Comment:
It sets the scope.zIndex to the literal value (@ takes the literal value) of mentio-z-index.